### ISSUE TITLE
PowerSupply pcs = 0 sets current to max

### ISSUE THREAD

**Issue by arkostin** [fossasia/pslab-python#167]
Thanks for taking the time to read this :)

Currently, setting the current on the PCS pin in the Power Supply instrument to 0 sets it to the maximum value (3.3e-3).

Here's some code to replicate the behavior:
```
from pslab import PowerSupply
ps = PowerSupply()

print(ps.pcs) # Prints 0
ps.pcs = 0
print(ps.pcs) # Prints 0.0033
```

Using Python 3.7.9, pslab 2.0.0

**Comment by bessman** [comment]
Thanks for reporting this. It will be fixed shortly. In the meantime, rest assured the bug is purely aesthetical; the actual current is zero.

**Comment by bessman** [comment]
Resolved by #168.
